__docs__/wonder-blocks-tokens/tokens-semantic-color.mdx

```diff
@@ -53,7 +53,7 @@ import {semanticColor} from "@khanacademy/wonder-blocks-tokens";
 const styles = {
     background: semanticColor.surface.secondary,
```

Comment:
On another note, do we want to deprecate the `surface` tokens in favour of the semantic core background color tokens too? I don't see them in the `deprecated` section in our docs, so I thought I'd check! https://khan.github.io/wonder-blocks/?path=/docs/foundations-using-color--docs#surface

Reply:
@jandrade WB-2041 is for the `status` tokens! I know we're also discussing with Caitlyn around if the `surface` tokens should be part of the `core` tokens, so it might be related to that too!